core/src/main/java/org/mskcc/cbio/oncokb/model/IndicatorQueryResp.java

```diff
@@ -1,42 +1,76 @@
 package org.mskcc.cbio.oncokb.model;
 
-
 import org.mskcc.cbio.oncokb.apiModels.Implication;
 import org.mskcc.cbio.oncokb.apiModels.MutationEffectResp;
 
+import io.swagger.annotations.ApiModelProperty;
+
 import java.util.ArrayList;
 import java.util.List;
 
-
-/**
- * TumorType generated by hbm2java
- */
-
 public class IndicatorQueryResp implements java.io.Serializable {
     private Query query;
-    private Boolean geneExist;
-    private Boolean variantExist;
-    private Boolean alleleExist;
+
+    @ApiModelProperty(value = "Indicates whether the gene is an OncoKB cancer gene")
+    private Boolean geneExist = false;
+
+    @ApiModelProperty(value = "Indicates whether an exact match for the queried variant is curated")
+    private Boolean variantExist = false;
+
+    @ApiModelProperty(value = "Indicates whether the alternate allele has been curated. See SOP Protocol 9.1")
+    private Boolean alleleExist = false;
+
+    @ApiModelProperty(value = "The oncogenicity status of the variant. Defaulted to \"Unknown\".", allowableValues = "Oncogenic, Likely Oncogenic, Likely Neutral, Inconclusive, Resistance, Unknown")
     private String oncogenic;
+
     private MutationEffectResp mutationEffect;
+
+    @ApiModelProperty(value = "(Nullable) The highest sensitivity level from a list of therapeutic evidences.", allowableValues = "LEVEL_1, LEVEL_2, LEVEL_3A, LEVEL_3B, LEVEL_4")
```

Comment:
Maybe "The highest sensitivity level from the list of therapeutic evidences saved in OncoKB"?

Reply:
I would say all the data in oncokb that is available is already saved, so I would omit that.